editor/plugins/theme_editor_plugin.cpp

```diff
@@ -3333,6 +3333,9 @@ void ThemeTypeEditor::_update_stylebox_from_leading() {
 }
 
 void ThemeTypeEditor::_type_variation_changed(const String p_value) {
+	//TODO: fix it so that when you clear type variation, you remove the old docs
```

Comment:
Isn't it done below? (with `remove_doc()`)

Reply:
It does remove it actually, this comment is a bit misleading. There's a caching issue in ```EditorHelpBit::HelpData EditorHelpBit::_get_theme_item_help_data``` in the way doc_property_cache is currently handled. Right now if there's a caching collision where the base type is changed, but you hover over a property shared between the previous base class and the new one, it loads up the previous base class description. So the type variation is cleared, it just doesn't appear to be in those cases